CHANGELOG.md

```diff
@@ -6,11 +6,24 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+- **CUMULUS-2203**
```

Comment:
Seems like these changes should be under a header for 3.0.1?

Reply:
That's in the release PR: https://github.com/nasa/cumulus/pull/1952

Reply:
Note 2200 should have been in the 3.0.0 release info, I'm going to update the master CL as well as the release notes.